x-pack/plugins/observability_solution/investigate_app/server/services/create_investigation.ts

```diff
@@ -23,7 +23,7 @@ export async function createInvestigation(
     ...params,
     updatedAt: now,
     createdAt: now,
-    createdBy: user.username,
+    createdBy: user.profile_uid!,
```

Comment:
```suggestion
    createdBy: user.profile_uid ?? user.username,
```

i wonder if it makes sense